packages/ast-spec/src/base/ClassBase.ts

```diff
@@ -0,0 +1,73 @@
+import type { Identifier } from '../expression/Identifier/spec';
+import type { ClassBody } from '../special/ClassBody/spec';
+import type { Decorator } from '../special/Decorator/spec';
+import type { TSClassImplements } from '../special/TSClassImplements/spec';
+import type { TSTypeParameterDeclaration } from '../special/TSTypeParameterDeclaration/spec';
+import type { TSTypeParameterInstantiation } from '../special/TSTypeParameterInstantiation/spec';
+import type { LeftHandSideExpression } from '../unions/LeftHandSideExpression';
+import type { BaseNode } from './BaseNode';
+
+export interface ClassBase extends BaseNode {
+  /**
+   * `true` if the class is an abstract class.
+   * ```
+   * abstract class Foo {...}
+   * ```
+   * This is always `undefined` for `ClassExpression`.
+   */
+  // TODO - make this `false` if it is not `abstract`
```

Comment:
Are these TODOs tracked anywhere? I think it'd be nice to include an issue number for all of them to encourage folks to get involved with the repo.

Reply:
I've tracked them whereever there's a `TODO(#issue_number)` - otherwise I haven't filed anything.
I'll file an issue for the remaining ones. Most of these TODOs are breaking changes I was going to action all at once for the next major.